CHANGELOG.md

```diff
@@ -7,8 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## Changed
```

Comment:
Looks like a bad merge and changed section repeated, remove this one

Reply:
[bb74cd2](https://github.com/ar-io/arns-react/pull/680/commits/bb74cd275603ce9e22479aa002faeef9d1035c40)